src/hastings_rpc.erl

```diff
@@ -109,10 +109,12 @@ get_update_seq(DbName) ->
 
 
 get_or_create_db(DbName, Options) ->
+    DbOpts = mem3_shards:opts_for_db(DbName),
+    Options1 = mem3_util:merge_opts(DbOpts, Options),
     case couch_db:open_int(DbName, Options) of
```

Comment:
Should we use  `couch_db:open_int(DbName, Options1)`?

Reply:
just referencing this comment https://github.com/apache/couchdb/pull/3633#discussion_r654376455
for possibly moving the creation logic within mem3

Reply:
Correct and thanks! 👍